streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractProcessorContext.java

```diff
@@ -246,4 +246,4 @@ public TaskType taskType() {
     public String changelogFor(final String storeName) {
         return stateManager().changelogFor(storeName);
     }
-}
+}
```

Comment:
I don't understand what this change is

Reply:
It's just removing a newline at the end. It's harmless.